### ISSUE TITLE
[solved] Error: No ResourceLoader implementation has been provided. Can't read the url "app.component.html"

### ISSUE THREAD

**Issue by tuxado** [angular/universal-starter#353]
I have this error i dont know why??? can help

```
`import { Component, Directive, ElementRef, Renderer, ChangeDetectionStrategy, ViewEncapsulation } from '@angular/core';

@Component({
  changeDetection: ChangeDetectionStrategy.Default,
  encapsulation: ViewEncapsulation.Emulated,
  selector: 'app',
  templateUrl : './app.component.html'
})
export class AppComponent {
}
`
```

```
`  module: {
    rules: [
      // TypeScript
      { test: /\.ts$/,   use: ['awesome-typescript-loader', 'angular2-template-loader'] },
      { test: /\.html$/, use: 'raw-loader' },
      { test: /\.css$/,  use: 'raw-loader' },
      { test: /\.json$/, use: 'json-loader' },
	  {
            test: /\.scss$/,
            use: [{
                loader: "raw-loader" // raw
            }, {
                loader: "style-loader" // creates style nodes from JS strings
            }, {
                loader: "css-loader" // translates CSS into CommonJS
            }, {
                loader: "sass-loader" // compiles Sass to CSS
            }],
			exclude: /node_modules/
       },
       {
          test: /initial\.scss$/,
          use: ExtractTextPlugin.extract({
            fallback: 'style-loader',
            use: 'css-loader!sass-loader?sourceMap'
          }),
			exclude: /node_modules/
       },

        {
          test: /\.woff(2)?(\?v=.+)?$/,
          use: 'url-loader?limit=10000&mimetype=application/font-woff',
			exclude: /node_modules/
        },

        {
          test: /\.(ttf|eot|svg)(\?v=.+)?$/,
          use: 'file-loader',
			exclude: /node_modules/
        },
        /* File loader for supporting images, for example, in CSS files.
         */
        {
          test: /\.(jpg|png|gif)$/,
          use: 'file-loader',
		  exclude: /node_modules/
        }
    ],
  },`

```
error:

```
`[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: C:\Server\public\dev\Server\dist/**/* src/index.html
[nodemon] starting `node dist/server/index.js`
C:\Server\public\dev\Server\dist\server\index.js:53810
        throw new Error("No ResourceLoader implementation has been provided. Can't read the url \"" + url + "\"");
        ^

Error: No ResourceLoader implementation has been provided. Can't read the url "app.component.html"
    at Object.get (C:\Server\public\dev\Server\dist\server\index.js:53810:15)
    at DirectiveNormalizer.module.exports.DirectiveNormalizer._fetch (C:\Server\public\dev\Server\dist\server\index.js:21087:43)
    at DirectiveNormalizer.module.exports.DirectiveNormalizer.normalizeTemplateAsync (C:\Server\public\dev\Server\dist\server\index.js:21130:21)
    at DirectiveNormalizer.module.exports.DirectiveNormalizer.normalizeDirective (C:\Server\public\dev\Server\dist\server\index.js:21105:44)
    at RuntimeCompiler.module.exports.RuntimeCompiler._createCompiledTemplate (C:\Server\public\dev\Server\dist\server\index.js:35976:167)
    at C:\Server\public\dev\Server\dist\server\index.js:35904:41
    at Array.forEach (native)
    at C:\Server\public\dev\Server\dist\server\index.js:35900:48
    at Array.forEach (native)
    at RuntimeCompiler.module.exports.RuntimeCompiler._compileComponents (C:\Server\public\dev\Server\dist\server\index.js:35899:43)
[nodemon] app crashed - waiting for file changes before starting...`
```

**Comment by tuxado** [comment]
Solved 

Might seem like a stupid post ( and it probably is ) but for those of you getting the " No ResourceLoader implementation has been provided " error while using the default starter kit with webpack and angular2-template-loader make sure that you don't put any white space between templateUrl and colon

Example :
templateUrl : './sometemplate.component.html' <- not good
templateUrl: './sometemplate.component.html' <- good

**Comment by MarkPieszak** [comment]
Logged an issue with angular2-template-loader Here https://github.com/TheLarkInn/angular2-template-loader/issues/57

We can close this out here since it's not universal related.

**Comment by tuxado** [comment]
update the version of the angular2-template-loader package from 0.4.0 to ^0.6.2 on the package.json file of the universal-starter starter project and use 

> npm install

 or :

```
npm remove angular2-template-loader
npm install angular2-template-loader@0.6.2 --save-dev
```

**Comment by MarkPieszak** [comment]
Hey we've updated this starter to include a `CLI` and a standalone webpack example using the latest `@angular/platform-server` that's been moved into Core. Closing this out since it's relating to the previous version, angular2-universal, but feel free to open a new issue if there's still an issue with the new version! Thanks.
